flink-runtime/src/test/java/org/apache/flink/runtime/AbstractIDTest.java

```diff
@@ -23,8 +23,8 @@
 import static org.junit.Assert.fail;
 
 import org.junit.Test;
-
 import org.apache.flink.runtime.testutils.CommonTestUtils;
+import org.apache.flink.util.AbstractID;
```

Comment:
Let us move this test to `flink-core`, with `AbstractID` being now part of the core.